frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -913,41 +942,232 @@ function ReaderFooter:addToMainMenu(menu_items)
                     self.settings.disable_progress_bar = not self.settings.disable_progress_bar
                     self:setTocMarkers()
                     self:refreshFooter()
-                    if self.settings.progress_bar_separate_line then
+                    if self.settings.progress_bar_separate_line or self.settings.progress_bar_bottom then
                         self.ui:handleEvent(Event:new("SetPageBottomMargin", self.view.document.configurable.b_page_margin))
                     end
                     UIManager:setDirty(nil, "ui")
                 end,
             },
             {
-                text = _("Show chapter markers"),
-                checked_func = function()
-                    return self.settings.toc_markers
+                text_func = function()
+                    local pos = _("aside")
```

Comment:
I find the whole "aside" thing highly cryptic.

I'm still not quite sure what it means in this context. Is that supposed to mean how it's always been (i.e., next to the text/icon stuff?).

Reply:
That's indeed what I meant (on the same line with the text/icon stuff) when suggesting it with my bad english (well, _aside_ sounded like a snobbish version of "on the side" to me :)
What's the correct word?

Reply:
Yeah, that's kind of my point: I took it to mean show me a *vertical* bar on one side of the screen ;).

Reply:
Text alongside/next to icons?

Reply:
I'd somewhat take a different approach, and use something based on the *variable* names, actually.

i.e., "Top", "Bottom", and "Bottom, on its own".

Reply:
Above footer items / Alongside footer items / Below footer items ?

Reply:
Oh, wait, the "Top" means above the items, not the top *of the screen* (again :D).

Then, yeah, what you just said ;p.